doc/conf.py

```diff
@@ -649,3 +672,15 @@ def linkcode_resolve(domain, info):
                 f"/{tag}/lib/{fn}{linespec}")
 else:
     extensions.append('sphinx.ext.viewcode')
+
+
+# -----------------------------------------------------------------------------
+# Sphinx setup
+# -----------------------------------------------------------------------------
+def setup(app):
+    if any(st in version for st in ('post', 'alpha', 'beta')):
```

Comment:
I think "post" would still be considered release?

Reply:
That's a good question, but I just moved this code to the bottom of the file. It looks like that's how it was when it came in, https://github.com/matplotlib/matplotlib/pull/13870/commits/310b004682551644bdcb90de9e3f9bc13974a1eb so maybe @tacaswell knows why

Reply:
I think we used to use `.postN` to count the number of commits past the last tag, but we switch to using. `.dev` for the number of commits on a pre-release of our expected next tag...but I think that means we need to add `'dev'` to this list as well?